ustriage/task.py

```diff
@@ -163,8 +222,36 @@ def short_title(self):
         }[self.obj.target.resource_type_link]
         return ' '.join(self.title.split(' ')[start_field:]).replace('"', '')
 
-    def get_releases(self, open_bug_statuses):
-        """List of chars reflecting per release status.
+    def __is_in_unapproved(self, series):
+        """Determine if this task is in a -unapproved for a series."""
+        # Thanks to Rbasak for the code that inspired this
+        ubuntu = Task.LP.distributions["ubuntu"]
+        distro_seriess = [ubuntu.getSeries(name_or_version=series)]
```

Comment:
nitpick: when I first looked at this one, I thought it was a typo. We name it distro_series_list or something similar to avoid further mistakes with these vars.

Reply:
This is from Robies code and follows his argument, therefore I'd keep it as-is unless strongly objecting.